src/aap_eda/core/tasking/__init__.py

```diff
@@ -162,6 +184,19 @@ def __init__(
         super().__init__(id, connection, serializer)
 
 
+# django-rq's rqworker command does not support --connection-class so
+# we cannot specify the DAB redis client that way.  Even if it did we
+# couldn't use it as DAB requires a url parameter that Redis does not.
+# If the connection a worker is given is not from DAB we replace it
+# with one that is.
+def _get_necessary_client_connection(connection: Connection) -> Connection:
+    if type(connection) not in [DABRedis, DABRedisCluster]:
```

Comment:
Is there a reason to not use `isinstance` instead of `type`?

Reply:
@Alex-Izquierdo No.  It would be my first choice but then I saw there is some EDA code that uses type().